packages/atomic/src/components/atomic-component-error/atomic-component-error.tsx

```diff
@@ -9,6 +9,10 @@ export class AtomicComponentError {
   @Prop() element!: HTMLElement;
   @Prop() error!: Error;
 
+  connectedCallback() {
+    console.error(this.error, this.element);
```

Comment:
Is it intended to check in this log statemented?

Reply:
Yes it's more descriptive than what is rendered, I figured seeing both is good